specification/regionmove/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/regionmovecollection.json

```diff
@@ -1662,19 +1676,63 @@
           "$ref": "#/definitions/ResourceSettings"
         }
       ],
-      "properties": {},
+      "properties": {
+        "enableDdosProtection": {
+          "description": "Gets or sets a value indicating whether gets or sets whether the\r\nDDOS protection should be switched on.",
+          "type": "boolean",
+          "x-nullable": true
+        },
+        "addressSpace": {
```

Comment:
For semantic considerations, maybe this makes more sense if it is plural? For instance `addressSpaces`

Reply:
We have aligned here with the network model as per the Microsoft.Network RP.

Refer: https://docs.microsoft.com/en-us/rest/api/virtualnetwork/virtualnetworks/get#addressspace